examples/flutter_gallery/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
 
   flutter:
     sdk: flutter
+  url_launcher: ^0.3.1
```

Comment:
With the package not being 1.0.0, a version constraint like ^0.3.1 would mean '>=0.3.1 <0.4.0', and this resolution would give the old version if we publish a 0.4.0.

But maybe it would make sense to move these packages to 1.0.0?